pkgs/top-level/all-packages.nix

```diff
@@ -14433,6 +14433,7 @@ with pkgs;
   wireguard-go = callPackage ../tools/networking/wireguard-go { };
 
   wkhtmltopdf-bin = callPackage ../tools/graphics/wkhtmltopdf-bin { };
+  wkhtmltopdf = wkhtmltopdf-bin;
```

Comment:
hydra doesn't allow aliases to be passed to packages.

we could rename all the inputs, or keep this manual alias here.

Reply:
#327890